src/callmodel.cpp

```diff
@@ -61,6 +60,57 @@ using namespace CommHistory;
  * ******** P R I V A T E   C L A S S   I M P L E M E N T A T I O N ********* *
  * ************************************************************************** */
 
+class CallModelPrivate : public EventModelPrivate
+{
+    Q_OBJECT
+    Q_DECLARE_PUBLIC( CallModel )
+
+public:
+    CallModelPrivate( EventModel *model );
+
```

Comment:
Actually, this is the verbatim pre-existing content of callmodel_p.h, moved into the implementation file.